php/msmtprc

```diff
@@ -0,0 +1,10 @@
+account default
```

Comment:
Could you add some comment to make sure users know this is only for Docker and not with a regular installation.

Reply:
Sure thing